### PR TITLE
Creates a servlet that tells a user if they are logged in or not.

### PR DESCRIPTION
**Pull request by meagancheese**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/LogInServlet.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+@WebServlet("/login")
+public class LogInServlet extends HttpServlet {
+  
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    response.setContentType("text/html");
+
+    UserService userService = UserServiceFactory.getUserService();
+    if(userService.isUserLoggedIn()){
```
Comment (GrantComm) [inline portfolio/src/main/java/com/google/sps/servlets/LogInServlet.java]: Add a space between the if and the opening parenthesis and a space between the closing parenthesis and the opening bracket per the java style guide

**Review by GrantComm — COMMENTED** [review]
Besides the spacing in the if conditional, LGTM

**Review by GrantComm — APPROVED** [review]

File: portfolio/pom.xml [hunk 2]
```diff
@@ -33,6 +33,16 @@
       <artifactId>appengine-api-1.0-sdk</artifactId>
       <version>1.9.59</version>
     </dependency>
+    <dependency>
+      <groupId>com.google.appengine</groupId>
+      <artifactId>appengine-api-1.0-sdk</artifactId>
+      <version>1.9.59</version>
+    </dependency>
+    <dependency>
```
Comment (acmcarther) [inline portfolio/pom.xml]: Duplicate dep?

**Review by acmcarther — APPROVED** [review]
IIUC, this PR does not add any way to log in, so it's not possible to *be* logged in, right? If so, how did you test this?

**Review by apluscs — APPROVED** [review]
LGTM

**Comment by meagancheese** [comment]
Some of the examples in the walkthrough file allowed you to log in and out and it carried over to my portfolio because it was being run on the same server.

File: portfolio/pom.xml (hunk 2 above)
Comment (acmcarther) [reply portfolio/pom.xml]: Actually this is *double* duplicated. It's already present before the change

Was this added by some tool?

**Pull request merged**